roles/storage-cinder/tasks/deprovision.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Delete Service Account
+  command: kubectl delete serviceaccount {{service_account}} -n {{namespace}}
+  when: sa.stdout == ""
+
+- name: Delete {{privileged_service_account}} serviceaccount
+  command: kubectl create serviceaccount {{privileged_service_account}} -n {{ namespace }}
+  when: psa.stdout == ""
+
+- name: Select a host for Mariadb hostpath
+  command: kubectl get nodes --selector="{{infra_node_label}}"=true
+  register: infra_node_hostnames
+
+- name: Remove label on the nodes that host Mariadb
+  command: kubectl label node "{{ item }}" "{{infra_node_label}}"-
+  with_items: "{{ infra_node_hostnames.stdout.split() }}"
+
+- name: Include  Rabbitmq
```

Comment:
If you write RabbitMQ in README, write it that way everywhere in docstrings, please. The same for MariaDB.

Reply:
Ack. Changed it for consistence. Thanks for catching this.